Geometry/HGCalCommonData/src/HGCalCellOffset.cc

```diff
@@ -0,0 +1,138 @@
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "Geometry/HGCalCommonData/interface/HGCalCellOffset.h"
+#include <vector>
+#include <iostream>
+
+//#define EDM_ML_DEBUG
+
+HGCalCellOffset::HGCalCellOffset(
+    double waferSize, int32_t nFine, int32_t nCoarse, double guardRingOffset_, double mouseBiteCut_) {
+  ncell_[0] = nFine;
+  ncell_[1] = nCoarse;
+  hgcalcell = std::make_unique<HGCalCell>(waferSize, nFine, nCoarse);
```

Comment:
Is this variable used somewhere?
I see it's a data member! Then please follow the usual convention and mark it explicitly using the `_` convention you used for other data members.

Reply:
done